packages/grafana-ui/src/components/FileDropzone/FileListItem.test.tsx

```diff
@@ -45,7 +45,7 @@ describe('The FileListItem component', () => {
     const abortUpload = jest.fn();
     render(<FileListItem file={{ file: file({}), id: '1', error: null, progress: 6, abortUpload }} />);
 
-    fireEvent.click(screen.getByRole('button', { name: 'Cancel' }));
+    fireEvent.click(screen.getByRole('button', { name: 'Cancel upload' }));
```

Comment:
If you want the tests to be a bit more robust, you can use regex to match text in this particular case (probably not a good idea with `*ByText` query as it may match unrelated elements.). This way you won't need to update test when there's some minor UI changes.
```suggestion
    fireEvent.click(screen.getByRole('button', { name: /cancel/i }));
```